docs/source/conf.py

```diff
@@ -26,8 +29,9 @@
 author = 'Edoardo Pasca'
 
 # The short X.Y version
-version = re.sub('^v', '', os.popen('git describe').read().strip())
+version = version.version

```

Comment:
I wonder which one is to be preferred: getting the version string from the code or from git.
Why did you choose the second after the first?

Reply:
I changed it because with git it didn't work! I got an error:
` /bin/sh: 1: git: not found`
I haven't worked out why, or how to fix it which is why I changed it. This is a problem in the iDVC repo because currently in that one we don't install the module, so can't use version.py